spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -1,20 +1,50 @@
 # frozen_string_literal: true
 
+require 'byebug'
+require 'rails_helper'
+
 RSpec.describe Api::V1::KeywordsController, type: :request do
   describe 'GET#index' do
-    it 'returns success status' do
-      get api_v1_keywords_path, headers: create_token_header
+    context 'given a logged in user gets theirs keyword' do
+      it 'returns success status' do
+        user = Fabricate(:user)
+        Fabricate.times(3, :keyword, user_id: user.id)
+
+        get api_v1_keywords_path, headers: create_token_header(user)
+
+        expect(response).to have_http_status(:success)
+      end
+
+      it 'returns three items' do
+        user = Fabricate(:user)
+        Fabricate.times(3, :keyword, user_id: user.id)
```

Comment:
Are we allow to fabricate with `user` instead of `user.id` as well?

Reply:
yup, that's much cleaner

```suggestion
        Fabricate.times(3, :keyword, user: user)
```

Reply:
As a side note: be aware that when building an instance (`Fabricate.build`) it will only assign the attributes that are declared. If you use `user: user`, it means the `user_id` will not be assigned. 
So depending on what your code is doing, you might need one or another. 

This should also be taken into consideration when writing the Fabricator. 

In Flexi, we ended up with the rule of thumb: 
- always use `instances` (user) — to ensure 99% of the cases will be consistent and to avoid duplicate declarations
- only use `foreign key id` (user.id) when there is no choice (e.g. a form that parses attributes expect ids, not instances).